src/SFA.DAS.EmployerAccounts.Api.UnitTests/Controllers/AccountPayeSchemesControllerTests/WhenIGetAPayeScheme.cs

```diff
@@ -4,11 +4,11 @@
 using FluentAssertions;
 using Moq;
 using NUnit.Framework;
-using SFA.DAS.EAS.Account.Api.Types;
-using SFA.DAS.EAS.Application.Queries.GetPayeSchemeByRef;
-using SFA.DAS.EAS.Domain.Models.Account;
+using SFA.DAS.EmployerAccounts.Api.Types;
+using SFA.DAS.EmployerAccounts.Models.Account;
+using SFA.DAS.EmployerAccounts.Queries.GetPayeSchemeByRef;
 
-namespace SFA.DAS.EAS.Account.Api.UnitTests.Controllers.AccountPayeSchemesControllerTests
+namespace SFA.DAS.EmployerAccounts.Api.UnitTests.Controllers.AccountPayeSchemesControllerTests
```

Comment:
This diff is misleading; looking in he branch I can see a completely new test class under `SFA.DAS.EmployerAccounts.Api.UnitTests.Controllers.AccountPayeSchemesControllerTests`